src/templates/MoreFavorites/MoreFavorites.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  RelatedSection,
+  RelatedBar,
+  RelatedBarTxt,
+  ScrollDiv,
+} from './MoreFavorites.styled';
+import RelatedVideosCard from '../RelatedVideosCard/RelatedVideos.details';
+
+import { useContext } from 'react';
+import { ThemeContext } from '../../providers/Theme/Theme.provider';
+
+function favStorage() {
+  const values = [];
+  const keys = Object.keys(localStorage);

```

Comment:
You should create the function in a `utils` file so that you can call ir from here and from the other file where you need it